tests/integration/security/testdata/authz/v1beta1-egress-gateway.yaml.tmpl

```diff
@@ -100,9 +100,9 @@ spec:
         port: 80
       route:
       - destination:
-          host: b.{{ .Namespace }}.svc.cluster.local
+          host: {{ .c }}.{{ .Namespace }}.svc.cluster.local
           port:
-            number: 8090
+            number: 8095
```

Comment:
why do we need to change the port? here and other places.

Reply:
`8090` is already allocated for `InstancePort` in https://github.com/istio/istio/blob/04c82fea7cf89f78bf4d679ef1bae00c7497e314/tests/integration/security/util/framework.go#L80.

I am reusing the existing workloads for this test and removing the ones created within this test.